### PR TITLE
Add users per page selector

### PR DESCRIPTION
**Pull request by landonreed**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing

### Description

Works with changes in https://github.com/ibi-group/datatools-server/pull/353. Add users per page selector to user list:

image

### CODE REVIEW

**Review by evansiroky — APPROVED** [review]

File: lib/admin/actions/admin.js [hunk 1]
```diff
@@ -47,6 +48,10 @@ export const setUserPage = createAction(
   'SET_USER_PAGE',
   (payload: number) => payload
 )
+export const setUserPerPage = createAction(
+  'SET_USER_PER_PAGE',
```
Comment (binh-dam-ibigroup) [inline lib/admin/actions/admin.js:52]: Echoing my suggestion from https://github.com/ibi-group/datatools-server/pull/353#discussion_r522168925 to rename `perPage` and `*_PER_PAGE` to `itemsPerPage` and `*_ITEMS_PER_PAGE`, respectively.

File: lib/admin/components/UserList.js [hunk 2]
```diff
@@ -3,7 +3,19 @@
 import Icon from '@conveyal/woonerf/components/icon'
 import React, {Component} from 'react'
 import {connect} from 'react-redux'
-import {Panel, Row, Col, Button, ButtonGroup, InputGroup, FormControl, ListGroup, ListGroupItem} from 'react-bootstrap'
+import {
+  Panel,
```
Comment (binh-dam-ibigroup) [inline lib/admin/components/UserList.js:7]: @evansiroky didn't request alphabetization on this one? :D

File: lib/admin/components/UserList.js [hunk 3]
```diff
@@ -132,6 +151,21 @@ class UserList extends Component<Props, State> {
...
+            <div className='pull-right'>
+              {this.messages('perPage')}{' '}
+              <DropdownButton
+                id='per-page-dropdown'
+                title={perPage}
+                onSelect={this._setUserPerPage}>
+                {// Render users per page options.
+                  [10, 25, 50, 100].map(val =>
+                    <MenuItem key={`option-${val}`} eventKey={val}>
+                      {val} {val === perPage ? <Icon type='check' /> : null}
```
Comment (binh-dam-ibigroup) [inline lib/admin/components/UserList.js:163]: Inconsistent with the feed version selector, where the check mark appears to the left of the selection.

**Review by binh-dam-ibigroup — APPROVED** [review]
Only minor things, but still good to go.

File: lib/admin/components/UserList.js (hunk 3 above)
Comment (landonreed) [reply lib/admin/components/UserList.js:163]: Yes consistent with the feed source 'sort by' dropdown in the project view... what to do...

File: lib/admin/actions/admin.js (hunk 1 above)
Comment (landonreed) [reply lib/admin/actions/admin.js:52]: Going to leave perPage in place to simplify the query param stringify, but happy to change the action name.

File: lib/admin/components/UserList.js (hunk 3 above)
Comment (landonreed) [reply lib/admin/components/UserList.js]: Lemme know what you think of bc0eff59ba7387aa53eb080f7d792fcae6093fa8

**Review by binh-dam-ibigroup — APPROVED** [review]
Please update the snapshots. Good idea to extract the menu item component.

**Pull request merged**